src/app/public/modules/overlay/overlay.component.scss

```diff
@@ -0,0 +1,41 @@
+:host {
+  pointer-events: none;
+}
+
+:host,
+.sky-overlay {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+}
+
+.sky-overlay-content {
+  position: relative;
+  z-index: 1;
+  display: inline-flex;
+  align-self: start;
+}
+
+.sky-overlay-ignore-pointer-events {
+  pointer-events: none;
+
+  ::ng-deep * {
+    pointer-events: auto;
+  }
+}
+
+.sky-overlay-backdrop {
+  background: rgba(0, 0, 0, .6);
```

Comment:
I noticed this alpha mask is darker than modals (`0.5`). Is that deliberate?

Reply:
Nope! Good catch.